[DOCU-2759] Updated Service Hub Overview & Service Overview (#5010)
Kong/docs.konghq.com: app/konnect/servicehub/manage-services.md

---

> title: Konnect Service Dashboard

Reviewer: I think this doc might need to be deleted since you combined the overview and dashboard. Right now it's unlisted but will show up in the Konnect docs.

---

The Konnect Service Package Dashboard is the place to manage services, versions, and documentation. The dashboard is available by clicking any Service Package in the Service Hub. 

> Here are some of the things you can do from the Service package Dashboard: 

Reviewer: Should it be "Service Package" here like it was in the first paragraph?

* Configure a service
* Publish a service to the Dev Portal
* Manage versions and labels. 
* View traffic, error, and latency data. 

![{{site.konnect_short_name}} service hub](/assets/images/docs/konnect/konnect-service-package-versions.png)

Number | Item | Description
-------|------|------------
1 | **Service Versions** | This section displays the status of a service version. From the context menu you can **Delete** a service, or use the **View Details** button to navigate to that versions dashboard. 
2 | **Analytics** | Analytics data for the service. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/)
3 | **Documentation** | You can add markdown documentation for your service. The documentation will render in the dashboard. If the service is published, the documentation will be available to developers in the Dev Portal. 

> ### Service Versions

Reviewer: ```suggestion
### Service versions
```

> A {{site.konnect_short_name}} service version is associated with a [runtime group](/konnect/runtime-manager/runtime-groups/). Configurations for the service version include configurations, plugins, specific implementations that are associated with the same runtime group. 

Reviewer: ```suggestion
A {{site.konnect_short_name}} service version is associated with a [runtime group](/konnect/runtime-manager/runtime-groups/). As such, the configurations, plugins, specific implementations that are associated with the runtime group are also associated with the service version. 
```
Just tried rewriting this to remove the repetition of "configurations".
